src/main/java/org/intellij/sdk/codesync/alerts/PricingAlerts.java

```diff
@@ -34,6 +34,20 @@ public static void setPlanLimitReached(Project project) {
         );
     }
 
+    public static void setPlanLimitReached(Project project, boolean privateRepoCountLimitReached) {
```

Comment:
We can rename this to `showPrivateRepoCountLimitReached`, we don't need any param here. 
We will do 
1. `getUserSubscription()` and call ```public PricingAlertDialog(Boolean canAvailTrial)``` with API response